docs/custom_virtualenvs.md

```diff
@@ -60,8 +67,9 @@ index aa8b304..eb05f91 100644
     fi
  
 +requirements_custom:
-+	virtualenv $(VENV_BASE)/my-custom-env
-+	$(VENV_BASE)/my-custom-env/bin/pip install python-memcached psutil
```

Comment:
Mistake here, fixing it. Should be `/opt/my-envs/my-custom-env/bin/pip install python-memcached psutil`